Extension/src/LanguageServer/settings.ts

```diff
@@ -203,6 +437,7 @@ export class OtherSettings {
     }
 
     public get editorTabSize(): number | undefined { return vscode.workspace.getConfiguration("editor", this.resource).get<number>("tabSize"); }
```

Comment:
I believe editorTabSize and editorInsertSpaces, and everything related to them, can be removed, as they are not used.  The right values to use for these are instead passed with every formatting request.